nixos/modules/services/backup/znapzend.nix

```diff
@@ -9,6 +9,26 @@ in
   options = {
     services.znapzend = {
       enable = mkEnableOption "ZnapZend daemon";
+
+      logLevel = mkOption {
+        default = "debug";
+        example = "warning";
+        type = lib.types.enum ["debug" "info" "warning" "err" "alert"];
+        description = "The log level when logging to file. Any of debug, info, warning, err, alert. Default in daemonized form is debug.";
+      };
+
+      logTo = mkOption {
+        type = types.str;
+        default = "";
```

Comment:
```
$ znapzend --logto=
Option logto requires an argument
```

This option should have a sensible default or appended optianlly like noDestroy.